frontend/src/pages/SocialPage/index.js

```diff
@@ -20,22 +19,22 @@ const SocialPage = () => {
 
   return (
     <div>
-      <Header>Subvisual</Header>
+      <Header>{constants.COMPANY_NAME}</Header>
       <Title>Thanks for playing!</Title>
 
       <div className={styles.subtitle}>
         <Text>{socialText}</Text>
       </div>
 
       <div className={styles.item}>
-        <SocialMediaButton name="github" username="subvisual" />
-        <SocialMediaButton name="linkedin" username="wearesubvisual" />
-        <SocialMediaButton name="behance" username="subvisual" />
-        <SocialMediaButton name="facebook" username="subvisual.co" />
-        <SocialMediaButton name="instagram" username="wearesubvisual" />
-        <SocialMediaButton name="dribbble" username="subvisual" />
-        <SocialMediaButton name="medium" username="subvisual" />
-        <SocialMediaButton name="twitter" username="subvisual" />
+        <SocialMediaButton name="github" />
+        <SocialMediaButton name="linkedin" />
+        <SocialMediaButton name="behance" />
+        <SocialMediaButton name="facebook" />
+        <SocialMediaButton name="instagram" />
+        <SocialMediaButton name="dribbble" />
+        <SocialMediaButton name="medium" />
+        <SocialMediaButton name="twitter" />
```

Comment:
Should this be a list? If so, you can use `ul` and the class name should be `items`.

Also, this looks like it could use a `map` or something like that–what do you think?